src/app/charts/charts-container.component.ts

```diff
@@ -17,7 +17,7 @@ import { ChartComponent } from './chart.component';
 })
 export class ChartsContainerComponent extends OnInit {
     private chartList: Array<String>;
-    private chartData: ChartData[];
+    private chartData: Array<ChartData>;
```

Comment:
There are a number of other places in the code defining things as `Thing[]`. Neither declaration form seem to displease the linter. Is there a TypeScript preference for the `Array<Thing>` syntax?